android/src/main/java/io/ionic/portals/reactnative/ReactNativePortalsModule.kt

```diff
@@ -213,6 +232,15 @@ internal class PortalViewManager(private val context: ReactApplicationContext) :
         setupLayout(parentView)
 
         val portalFragment = PortalFragment(portal)
+
+        val configBuilder = CapConfig.Builder(context)
+            .setInitialFocus(false)
```

Comment:
When I looked in ionic-portals-android, I saw that this was being set if there was no CapConfig defined. @carlpoole Do you think this should be added as a configuration to the Portal as well?

Reply:
Perhaps, but maybe we should look at the config as a whole and see if we should make some things more configurable on the Portal instead of reaching into the config to have to do some things